exporter/collector/logs.go

```diff
@@ -196,6 +196,17 @@ func (l *LogsExporter) Shutdown(ctx context.Context) error {
 }
 
 func (l *LogsExporter) PushLogs(ctx context.Context, ld plog.Logs) error {
+	err := l.pushLogs(ctx, ld)
+	if err != nil {
+		l.obs.log.Error(
+			"Exporting logs failed",
+			zap.Error(err),
+		)
+	}
```

Comment:
Should we make a generic wrapper for this kind of thing? Similar to how it worked with retry helper before

Reply:
We could write a util function, but it would need access to each logger anyways.  Without it being built into the component framework (e.g. like exporterhelper), I don't think we can write a generic wrapper.